src/ir/lowering.rs

```diff
@@ -1378,7 +1476,8 @@ fn lower_fn_call(
     // The id of the fn to call, in case its a method.
     fn_id: Option<DefId>,
 ) -> Result<(Rvalue, Ty, Span), LoweringError> {
-    let fn_id = {
+    debug!("lowering fn call");
```

Comment:
maybe we can add the Id of what fn we are lowering to help debugging.

Reply:
![image](https://github.com/user-attachments/assets/b667442a-cf8f-4fb2-b8aa-d7e79525ed52)

the instrument scope adds this info